docs/root/start/start.rst

```diff
@@ -12,222 +12,16 @@ These examples use the :ref:`v3 Envoy API <envoy_api_reference>`, but use only t
 feature of the API, which is most useful for simple requirements. For more complex requirements
 :ref:`Dynamic Configuration <arch_overview_dynamic_config>` is supported.
 
-.. _install:
-
-Building and installation
--------------------------
-
 .. toctree::
-    :maxdepth: 2
+    :maxdepth: 3
 
+    install
+    quick-start
+    docker
+    tools/tools
```

Comment:
In looking at this now, WDYT about moving tools under operations and administration? I think it's a bit in the weeds for the getting started guide?

Reply:
wfm